py3-ml-metadata.yaml

```diff
@@ -49,6 +50,9 @@ pipeline:
     with:
       series: series
 
+  - runs: |
+      touch /etc/gcc_spec_file
```

Comment:
i think i would prefer Setting the environment var to /dev/null.
I'm interested in your thoughts on that versus this.

Reply:
Doesn't work which is why we have I'm making the file. The build eventually clears all the environment variables.

Reply:
Which is to say I also would've preferred the environment variable.